templates/ovncontroller/bin/functions

```diff
@@ -52,31 +52,59 @@ function configure_external_ids {
 # Configure bridge mappings and physical bridges
 function configure_physical_networks {
     local OvnBridgeMappings=""
+    local br_new=""
     for physicalNetwork in ${PhysicalNetworks}; do
         br_name="br-${physicalNetwork}"
-        ovs-vsctl --may-exist add-br ${br_name}
-        ovs-vsctl --may-exist add-port ${br_name} ${physicalNetwork}
         bridgeMapping="${physicalNetwork}:${br_name}"
         if [ -z "$OvnBridgeMappings"]; then
             OvnBridgeMappings=$bridgeMapping
+            br_new=$br_name
         else
             OvnBridgeMappings="${OvnBridgeMappings},${bridgeMapping}"
+            br_new="${br_new} ${br_name}"
+        fi
+    done
+
+    # Current configured bridges.
+    ovn_bms=$(ovs-vsctl --if-exists get open . external_ids:ovn-bridge-mappings|tr -d '"')
+    local br_current=""
+    for bm in ${ovn_bms/,/ }; do
+        if [ -z "$br_current"]; then
+            br_current=${bm##*:}
+        else
+            br_current="${br_current} ${bm##*:}"
         fi
```

Comment:
seems it no longer set or remove external_ids:ovn-bridge-mappings

Reply:
Right